pkgs/applications/video/corrscope/default.nix

```diff
@@ -0,0 +1,53 @@
+{ mkDerivationWith
+, python3Packages
+, lib
+, wrapQtAppsHook
+, qtbase
+, ffmpeg
```

Comment:
it would be nice if this argument list was sorted.

Reply:
Sorted by what, alphabetically? It's sorted by usage: expression-only -> nativeBuildInputs -> buildInputs. I wouldn't mind sorting them alphabetically within those groups but sorting all of them alphabetically seems silly to me.

Reply:
ok, then sort the alphabetically per group.

Reply:
Done.